packages/gatsby-source-drupal/src/utils.js

```diff
@@ -271,23 +299,9 @@ ${JSON.stringify(nodeToUpdate, null, 4)}
           )
       }
     })
-
-    // see what nodes are newly referenced, and make sure to call `createNode` on them
-    const addedReferencedNodes = _.difference(
```

Comment:
This code isn't needed as it just duplicates what handleReferences does. We now return nodes with updated backreferences from that function so we can eliminate this code.